src/core/thermostat.cpp

```diff
@@ -209,8 +247,17 @@ void thermo_heat_up()
 {
   int j;
   if(thermo_switch & THERMO_LANGEVIN) {
+#ifndef PARTICLE_ANISOTROPY
     langevin_pref2_buffer          = langevin_pref2;
     langevin_pref2 *= sqrt(3);
```

Comment:
Just for the records: the Espresso variant of Espresso reuses forces in the Verlet algorithm - usually. Thus, you add two times the same randomly drawn number, and your variance is in fact 4*s, where s is the desired variance. This compensated for in the Langevin prefactor, if you ever wonder where the 24.0 in the prefactor comes from.
However, in the first step of the integration loop, Espresso may need to recompute forces (recalc_forces, e.g. because a particle was moved). In that case, we get two different random numbers of variances s' and s. To get the same variance 4s of the sum, you need to choose s' = 3s, or a prefactor of sqrt(3) to the actual random number.